### ISSUE TITLE
Please support MSYS2 MINGW64

### ISSUE THREAD

**Issue by iahung** [MorcilloSanz/RendererGL#64]

**Comment by MorcilloSanz** [comment]
At the moment it has only been tested in Linux with GNU Compiler and in Windows with MSVC Compiler, working correctly in both. It is still at an early stage, but when the first official version is released, there should be no problem in compiling the code as a library and using it in the most common compilers.
Currently no testing has been done with MINGW, but the dependencies used should be compatible.

Best regards

**Comment by iahung** [comment]
> At the moment it has only been tested in Linux with GNU Compiler and in Windows with MSVC Compiler, working correctly in both. It is still at an early stage, but when the first official version is released, there should be no problem in compiling the code as a library and using it in the most common compilers. Currently no testing has been done with MINGW, but the dependencies used should be compatible.
> 
> Best regards

You should use the system's GLEW.

```
$ ninja
[1/37] Building CXX object CMakeFiles/RendererGL.dir/glew/glew.cpp.obj
FAILED: CMakeFiles/RendererGL.dir/glew/glew.cpp.obj
C:\msys64\mingw64\bin\c++.exe   -std=gnu++17 -MD -MT CMakeFiles/RendererGL.dir/glew/glew.cpp.obj -MF CMakeFiles\RendererGL.dir\glew\glew.cpp.obj.d -o CMakeFiles/RendererGL.dir/glew/glew.cpp.obj -c C:/msys64/home/Administrator/RendererGL/glew/glew.cpp
C:/msys64/home/Administrator/RendererGL/glew/glew.cpp:53:12: fatal error: wglew.h: No such file or directory
   53 | #  include <wglew.h>
      |            ^~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
```

**Comment by MorcilloSanz** [comment]
Yes, I am aware of that. Once the code is ready to release the first version, this kind of things will be fixed and a set of tests will be done to verify that everything works correctly 👍🏼
